### PR TITLE
feat: update mqtt doc

### PR DESCRIPTION
**Pull request by eeff**

### CODE REVIEW

File: directory.json [hunk 1]
```diff
@@ -314,8 +314,16 @@
           "title": "MQTT",
           "children": [
             {
-              "title": "MQTT 介绍及使用",
-              "path": "north-apps/mqtt/mqtt"
+              "title": "盖览",
```
Comment (gc87) [inline directory.json:317]: 概览

Comment (eeff) [reply directory.json]: Fix

File: zh_CN/north-apps/mqtt/api.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+以下内容描述 Neuron MQTT 插件如何上报采集的数据，以及如何通过 MQTT 协议读写点位数据。
+
+注意下文中涉及的 MQTT 主题中出现的 **{node_name}** 指代的是实际的 MQTT 北向节点名字。
+
+
+## 上传数据
+
+Neuron MQTT 插件将采集到的数据以 JSON 形式发布到 **upload-topic** 参数指定的主题,
+通过控制面板配置时默认为 **/neuron/{node_name}/upload** 。
```
Comment (lixiumei123) [inline zh_CN/north-apps/mqtt/api.md:11]: No spaces before 。

File: zh_CN/north-apps/mqtt/api.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+上报数据的具体格式由 **format** 参数指定，有 *tags-format* 和 *values-format* 两种格式。
+
+### Tags 格式
+
+以下为使用 *tags-format* 格式的数据样例，其中所有点位数据存放在一个数组中，每个数组元素包含
+点位的名字，采集成功时的数据值或者采集失败时的错误码。
```
Comment (lixiumei123) [inline zh_CN/north-apps/mqtt/api.md:18]: No spaces before “点”

File: zh_CN/north-apps/mqtt/api.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+      "error": 2014
+    }
+  ]
+}
+```
+
+### Values 格式
+
+以下为使用 *values-format* 格式的数据样例，采集成功的点位数据值存放在一个字典中，
+采集失败的点位错误码存放在另一个字典中。
```
Comment (lixiumei123) [inline zh_CN/north-apps/mqtt/api.md:41]: No spaces before “采”

File: en_US/north-apps/mqtt/api.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+
+### Request
+
+You can read a group of tags by sending requests in JSON to the MQTT topic
+**/neuron/{node_name}/read/req**.
+
+#### Body
+
+The read request body should have the following fields:
+* "uuid": a unique identifier, which will be echoed back in the response to help identify the corresponding request
```
Comment (lixiumei123) [inline en_US/north-apps/mqtt/api.md:78]: Pay attention to punctuation at the end of the sentence.

File: zh_CN/north-apps/mqtt/api.md [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+## 读 Tags
+
+### 请求
+
+通过发送 JSON 形式的请求到 MQTT 主题 **/neuron/{node_name}/read/req** ，您可以读取一组点位数据。
+
+#### 请求体
+
+读请求体由以下字段构成：
+* "uuid": 唯一的标志，会在响应中原样返回用以区分对应的请求
```
Comment (lixiumei123) [inline zh_CN/north-apps/mqtt/api.md:73]: Pay attention to punctuation at the end of the sentence.

Comment (eeff) [reply zh_CN/north-apps/mqtt/api.md:73]: Fix

File: zh_CN/north-apps/mqtt/api.md (hunk 3 above)
Comment (eeff) [reply zh_CN/north-apps/mqtt/api.md:18]: Fix

File: zh_CN/north-apps/mqtt/api.md (hunk 2 above)
Comment (eeff) [reply zh_CN/north-apps/mqtt/api.md:11]: Fix

File: directory.json [hunk 7]
```diff
@@ -314,8 +314,16 @@
           "title": "MQTT",
           "children": [
             {
-              "title": "MQTT 介绍及使用",
-              "path": "north-apps/mqtt/mqtt"
+              "title": "概览",
+              "path": "north-apps/mqtt/overview"
+            },
+            {
+              "title": "API",
```
[the remark this answers is not in the document]
Comment (eeff) [reply directory.json:321]: keep as "API"

**Pull request merged**